layouts/partials/scripts.html

```diff
@@ -1,11 +1,8 @@
-
-<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
-
-{{ if .Site.Params.mermaid.enable }}
-<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
-
-{{ end }}
+<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.6.1.min.js" integrity="sha384-VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2" crossorigin="anonymous"></script>
```

Comment:
What's the source for this file? (I can't get the subresource integrity check to come out right).

Reply:
Popper file.
```
curl -o popper-1.16.1.min.js https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js

~ create-sri popper-1.16.1.min.js
sha384 encryption result:
sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN

 ~ cat popper-1.16.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN%
```

Same file in ./static/js folder.
```
js git:(bugMermaid) create-sri popper-1.16.1.min.js
sha384 encryption result:
sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN

js git:(bugMermaid) cat popper-1.16.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN%
```
Code in `scripts.html`
```
<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
```

Reply:
Bootstrap file.
```
 ~ curl -o bootstrap-4.6.1.min.js https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js

 ~ cat bootstrap-4.6.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2%
 ```
 
 same file in ./static.js folder
 ```
 cat bootstrap-4.6.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2%
 ```
 Code in scripts.html
 ```
 <script src="/js/bootstrap-4.6.1.min.js" integrity="sha384-VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2" crossorigin="anonymous"></script>
 ```